### DISCUSSION TITLE
JS OS booteable

### DISCUSSION THREAD

**Discussion by studiosbeta** [os-js/OS.js#841] [Ideas]
We can convert turn js OS into a real operating system, using Linux as a base.

I think it is to create a linux distribution that starts a browser that opens js OS

something similar like Firefox OS

**Comment by andersevenrud** [comment]
Archived repo using Electron instead of a kiosk browser and Debian as base: https://github.com/os-js/osjs-debian

**Reply by KevDoy** [reply]
@andersevenrud I'm curious why this was archived. Is this purely to reduce maintenance/tech debt?

**Reply by andersevenrud** [reply]
@KevDoy It was just a proof of concept, really. And if my mind serves me correctly I made this for someone that I was in contact with and was put up on Github privately so it would not get lost, then changed it to be public because it might be of use to somebody.

**Reply by shortymc** [reply]
hi ! i wanna know how you put an html/js/css file on top of the os for make a user interface ?

**Reply by andersevenrud** [reply]
@shortymc What exactly do you mean by "make a user interface" ?

**Reply by shortymc** [reply]
i don't got the knowledge for make fusion, but just replace the original interface desktop with a website like user interface, like a webdesktop . i would like add it on the top of a linux distro like this an simple html/css/js file in place of the desktop.

**Reply by andersevenrud** [reply]
That is essentially what the linked repo does @shortymc.
